x-pack/plugins/cases/public/components/custom_fields/custom_fields_list/index.tsx

```diff
@@ -54,63 +50,59 @@ const CustomFieldsListComponent: React.FC<Props> = (props) => {
 
   const showModal = Boolean(selectedItem);
 
-  return (
+  return customFields.length ? (
     <>
       <EuiSpacer size="s" />
-      <EuiFlexGroup justifyContent="flexStart">
+      <EuiFlexGroup justifyContent="flexStart" data-test-subj="custom-fields-list">
         <EuiFlexItem>
-          {customFields.length ? (
-            <EuiDragDropContext onDragEnd={() => {}}>
```

Comment:
removed drag and drop as not used anymore

Reply:
Thanks for that!!